pwem/objects/data.py

```diff
@@ -152,7 +152,7 @@ def getRotationMatrix(self):
     def getEulerAngles(self):
         from pwem.convert.transformations import euler_from_matrix
         rotation = self.getRotationMatrix()
-        return euler_from_matrix(rotation)
+        return euler_from_matrix(rotation, 'szyz')
```

Comment:
May be better to be an argument "axes" with 'szyz' by default sois more versatile?